vercel.json

```diff
@@ -1184,6 +1184,11 @@
     "source": "/docs/concepts/console-manage-overview/",
     "destination": "https://www.sharetribe.com/help/en/collections/8975376-manage",
     "permanent": true
+  },
```

Comment:
There are a couple of articles that link to /how-to/set-up-and-use-stripe/
Those links could also be updated directly with the help center URL in order to avoid an extra redirect

Reply:
The articles in the codebase have already been updated with the help center URL – are there some other articles you're thinking of?